config/application.rb

```diff
@@ -216,6 +216,7 @@ module ::Reporting; end
     config.assets.precompile += ['web/all.js']
     config.assets.precompile += ['darkswarm/all.js']
     config.assets.precompile += ['shared/*']
+    config.assets.precompile += %w(mail.scss)
```

Comment:
For consistency, could you use the syntax to create an array as the rest  of the file ?